backend/src/pilcom_cli/json_exporter/mod.rs

```diff
@@ -34,7 +34,7 @@ pub fn export<T: FieldElement>(analyzed: &Analyzed<T>) -> JsonValue {
             StatementIdentifier::Definition(name) => {
                 if let (poly, Some(value)) = &analyzed.definitions[name] {
                     if poly.poly_type == PolynomialType::Intermediate {
-                        if let FunctionValueDefinition::Mapping(value) = value {
```

Comment:
so this was just buggy?

Reply:
I think so: it was treating `= x` the same as `(i) { x }` but this doesnt make sense from a type perspective: in the former `x` is a column (`int -> fe`) but in the latter (mapping used for fixed columns), x is a field element `fe`.